### PR TITLE
Install example command line tool as reporting-cli

### PR DESCRIPTION
**Pull request by cwasicki**
* Install command line tool as `reporting-cli`: The command line tool from the example is moved to the package and can be called via `reporting-cli` or `python -m frequenz.client.reporting` from the command line.
* The command-line argument `--display` is renamed to `--format` for `reporting-cli`.
* The `reporting-cli` tool can output CSV format, dataframe output is removed.

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -6,6 +6,12 @@
 
 ## Upgrading
 
+* Install command line tool as `reporting-cli`: The command line tool from the exampls is moved
```
Comment (Marenz) [inline RELEASE_NOTES.md]: exampls typo

Comment (cwasicki) [reply RELEASE_NOTES.md]: fixed

**Review by Marenz — APPROVED** [review]

**Comment by Marenz** [comment]
Just to mention it: @llucax was less thrilled about having the client in __main__, but for now I think it's fine.

**Comment by cwasicki** [comment]
> Just to mention it: @llucax was less thrilled about having the client in main, but for now I think it's fine.

Discussed with @llucax that we continue with this for now and follow-up on the structure subsequently.

**Pull request merged**